lively.morphic/text/morph.js

```diff
@@ -908,6 +913,16 @@ export class Text extends Morph {
     };
   }
 
+  setupSelectionColorCSSRuleForColor (color) {
+    const rules = document.styleSheets[0].cssRules;
+    let idx = 0;
+    for (const rule of rules) {
+      if (rule.selectorText === `#${this.id} > div > .line > span::selection`) document.styleSheets[0].deleteRule(idx);
+      idx++;
+    }
+    if (this.selectionMode === 'native') document.styleSheets[0].insertRule(`#${this.id} > div > .line > span::selection { background: #${color.toHexString()}; }`, 0);
```

Comment:
```suggestion
    if (this.selectionMode === 'native') document.styleSheets[0].insertRule(`#${this.id} > div > .line > span::selection { background: ${color.toCSSString()}; }`, 0);
```

Reply:
This will allow us to support transparent colors for the selection, which is often the case.